ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -43,9 +43,35 @@ class TabLocationView: UIView {
   let contentView = UIView()
   private var tabObservers: TabObservers!
   private var privateModeCancellable: AnyCancellable?
+  private var _url: URL?
 
   var url: URL? {
-    didSet {
+    get {
+      _url
+    }
+
+    set(newValue) {
+      _url = newValue
+
+      // Strip `user:pass` from blob URLs
+      if newValue?.scheme == "blob", let url = newValue,
```

Comment:
Will this code affect how non-blob URLs are displayed? We currently strip it from http/https schemes as well , just want to make sure we don't break it.

Reply:
No this affects only blob.
The part where it checks if the scheme is blob will make sure it only strips blob URLs.

I'm not sure if I've seen anything in our code that strips for http/https though.